example/speech-demo/io_util.py

```diff
@@ -300,7 +300,6 @@ def __init__(self, train_sets, batch_size, init_states, truncate_len=20, delay=5
         self.pad_zeros = pad_zeros
 
         self.time_major = time_major
-        self.layout_mapper = mx.io.DefaultLayoutMapper(1 if time_major else 0)
```

Comment:
I think we can merge with this for now, but maybe add a one line comment of TODO to change this to the new way.